go/pkg/tool/tool_test.go

```diff
@@ -109,55 +112,98 @@ func TestTool_CheckDeterminism(t *testing.T) {
 	e, cleanup := fakes.NewTestEnv(t)
 	defer cleanup()
 	cmd := &command.Command{
-		Args:        []string{"foo bar baz"},
+		Args:        []string{"foo", "bar", "baz"},
 		ExecRoot:    e.ExecRoot,
 		InputSpec:   &command.InputSpec{Inputs: []string{"i1", "i2"}},
 		OutputFiles: []string{"a/b/out"},
 	}
-	if err := os.WriteFile(filepath.Join(e.ExecRoot, "i1"), []byte("i1"), 0644); err != nil {
-		t.Fatalf("failed creating input file: %v", err)
-	}
-	if err := os.WriteFile(filepath.Join(e.ExecRoot, "i2"), []byte("i2"), 0644); err != nil {
-		t.Fatalf("failed creating input file: %v", err)
-	}
-	out := "output"
-	opt := &command.ExecutionOptions{AcceptCached: false, DownloadOutputs: true, DownloadOutErr: true}
-	_, acDg, _, _ := e.Set(cmd, opt, &command.Result{Status: command.SuccessResultStatus}, &fakes.OutputFile{Path: "a/b/out", Contents: out})
+	_, acDg, _, _ := e.Set(cmd, command.DefaultExecutionOptions(), &command.Result{Status: command.SuccessResultStatus}, &fakes.InputFile{Path: "i1", Contents: "i1"}, &fakes.InputFile{Path: "i2", Contents: "i2"}, &fakes.OutputFile{Path: "a/b/out", Contents: "out"})
 
 	client := &Client{GrpcClient: e.Client.GrpcClient}
 	if err := client.CheckDeterminism(context.Background(), acDg.String(), "", 2); err != nil {
 		t.Errorf("CheckDeterminism returned an error: %v", err)
 	}
-	// Now execute again with changed inputs.
```

Comment:
This comment was a result of copy-paste; inputs do not actually change, only output does.